src/opnsense/mvc/app/views/OPNsense/Diagnostics/log.volt

```diff
@@ -120,12 +128,12 @@
                 <div class="hidden">
                     <!-- filter per type container -->
                     <div id="severity_filter_container" class="btn-group">
-                        <select id="severity_filter"  data-title="{{ lang._('Severity') }}" class="selectpicker" multiple="multiple" data-width="200px">
+                        <select id="severity_filter"  data-title="{{ lang._('Maximum severity level') }}" class="selectpicker" data-width="200px">
                             <option value="Emergency">{{ lang._('Emergency') }}</option>
                             <option value="Alert">{{ lang._('Alert') }}</option>
                             <option value="Critical">{{ lang._('Critical') }}</option>
                             <option value="Error">{{ lang._('Error') }}</option>
```

Comment:
@kulikov-a you probably want to select error by default as well (maybe Critical and emergency too)

Reply:
but we only limit the maximum level? all levels below the specified (emerg..warning) one will be included in the request during `requestHandler` processing?

Reply:
@kulikov-a oops, missed the most important part in your commit. so that leaves the question if we do want to be able to filter specific levels or a set a cut off maximum. let me think about this for a while

Reply:
sure! thanks (I don't think you missed it, but I poorly expressed the idea)

Reply:
@kulikov-a I thought about this for a while, but I think we better keep the filters as is. If for example you're looking for a low priority message you would otherwise be forced to crawl through all higher level items as well. Which is ok for quite some use-cases, but could be annoying when looking for specific messages.

Reply:
@AdSchellevis I'm sorry, could you please explain in a little more detail? Perhaps you meant the opposite: will have to see messages with **less** severity? that is, if, for example, the administrator wants to see only Warnings messages and will also see Errors,  Alarms etc? I agree, this is possible.
Simply (and from my experience as an administrator) it seemed to me that in most cases (especially since we cannot know in advance the severity level of the required message) working with logs comes down to switching two modes: normal and debug (normal allows us to concentrate on messages about abnormal behavior , debug - get more information about what is happening in the system). additional filtering in this case can be performed by searching.
but perhaps you are right and I still need to compare to understand which of the methods is more convenient (now at 21.7.6 I use my patches. It's convenient for me)). thanks!

Reply:
@kulikov-a sure, if I'm looking for Informational messages, I would automatically receive Notices, Warnings and Errors too, which is a bit inconvenient. Making sure the default window opens with anything higher than warning is obviously fine, it just feels like a bit of a loss to not being able to filter on an exclusive level anymore.

Reply:
@AdSchellevis got it, thanks! if you would be ok with a mode switch checkbox (individual choice <-> maximum level), just let me know) I'll try to do that

Reply:
@kulikov-a that would be defendable indeed, wouldn't mind using maximum level by default in that case.

Reply:
great! I'll try )